geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb

```diff
@@ -160,7 +160,18 @@ The gateway sender configuration for a specific sender `id` must be identical on
 
 ## <a id="setting_up_a_multisite_system__section_E1DEDD0743D54831AFFBCCDC750F8879" class="no-quick-link"></a>Create Data Regions for Multi-site Communication
 
-When you are using a multi-site configuration, you choose which data regions to share between sites. Because of the high cost of distributing data between disparate geographical locations, not all changes are passed between sites. After you define gateway senders, configure regions to use the gateway sender to distribute region events.
+When you are using a multi-site configuration, you choose which data regions to share between sites. Because of the high cost of distributing data between disparate geographical locations, not all changes are passed between sites. 
+
+
+**Note these important restrictions on the regions:**
+
+- Replicated regions cannot use a parallel gateway sender. Use a serial gateway sender instead.
+
+- In addition to configuring regions with gateway senders to distribute events, you must configure the same regions in the target clusters to apply the distributed events. The region name in the receiving cluster must exactly match the region name in the sending cluster.
+
+- Regions using the same parallel gateway sender ID must be colocated.
+
+After you define gateway senders, configure regions to use the gateway sender to distribute region events.
```

Comment:
Is there a number mismatch here? Seems like it should be either "After you define a gateway **sender**" or "use the gateway **senders**."